src/main/java/net/sf/rails/algorithms/NetworkBuilder.java

```diff
@@ -39,53 +39,53 @@
 //
 //public class NetworkBuilder implements Iterable<NetworkVertex> {
```

Comment:
Should we perhaps remove this class @neutronc?

Reply:
Thats not really my code but @freystef  i really am not deep in that code there for the network calculation, route and revenue. I can debug it and am after years now familiar with the modifiers... But otherwise if you know something thats easier to read and understand and produces reliable results for network, route and revenue feel free :)

Reply:
FYI, this class/file is completely commented out. Maybe that is what @madoar is referring to?

Reply:
Exactly :)